### DISCUSSION TITLE
How to implement something equivalent of postgresql schemas through EdgeDB

### DISCUSSION THREAD

**Discussion by shah-dhwanil** [geldata/gel#4662] [Q&A]
I want to have something equvivalent postgresql schemas containing same EdgeDB types for each user.
 So how to implement it?

**Comment by elprans** [comment]
The equivalent of PostgreSQL schemas in EdgeDB are [`modules`](https://www.edgedb.com/docs/reference/sdl/modules#modules)

**Reply by shah-dhwanil** [reply]
@elprans Thanks for quick response.So I want to build a multi tenancy app using equivalent of schemas i.e modules in edgedb. So how do I make migration when new user sign ups  to our service.
